visualization/scripts/parse-data.js

```diff
@@ -57,7 +57,6 @@ let modelDirs = models.getModelDirs(
   '../data-processed',
   ['component-models']
 )
-
```

Comment:
Is there any changes in this file? Or you're just removing the empty line?

Reply:
Oops, yeah nice catch, there is no change to this file.